package/src/components/Attachment/ImageLoadingIndicator.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { ActivityIndicator, StyleSheet, View, ViewProps } from 'react-native';
+
+import { useTheme } from '../../contexts/themeContext/ThemeContext';
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    display: 'flex',
+    height: '100%',
+    justifyContent: 'center',
+    width: '100%',
+  },
+});
+
+export type ImageLoadingIndicatorProps = ViewProps;
+
+export const ImageLoadingIndicator = (props: ImageLoadingIndicatorProps) => {
+  const {
+    theme: {
+      messageSimple: {
+        videoThumbnail: { container },
```

Comment:
what is the idea behind getting the container styles from videoThumbnail here?

Reply:
The naming is wrong...

Reply:
This naming is not wrong. The styling for videoThumbnail is the exact styling I needed here. I was going to rename it to something a bit more generic but decided to let it be for now since that would be a breaking change @vanGalilea

Reply:
@Enigma-I-am let's just add a new one instead of videoThumbnail

Reply:
@vanGalilea this has been updated